pkg/s3wrapper/client.go

```diff
@@ -40,7 +40,7 @@ type API interface {
 	IsAwsS3() bool
 	CreateBucket() error
 	Upload(ctx context.Context, data []byte, objectName string) error
-	UploadStream(ctx context.Context, reader io.Reader, objectName string) error
+	UploadStream(ctx context.Context, reader io.Reader, objectName string, public bool) error
```

Comment:
"public" is ambiguous. Is the intention to make the stream that is uploaded public? If it is, then something like "makePubliclyVisible" would be more clear.

Reply:
Not relevant in new code